src/similarity.py

```diff
@@ -61,10 +72,9 @@ def modified_cosine(
 
     Returns
     -------
-    Tuple[float, Tuple[np.ndarray, np.ndarray]]
-        A tuple consisting of (i) the cosine similarity between both spectra,
-        and (ii) a tuple with arrays of the matching peak indexes in the first
-        and second spectrum, respectively.
+    Tuple[float, float, Tuple[np.ndarray, np.ndarray]]
```

Comment:
These docstrings don't match your new return value.

Reply:
I first wanted to wait for your comments to come to the final version before updating the docs